### ISSUE TITLE
`There is no formatter for 'hcl' files installed` when execute "Format Document"

### ISSUE THREAD

**Issue by fullheart** [hashicorp/vscode-terraform#1081]
## Versions

This bug is reproducible in:
 - [x] the **latest** version of the extension (below)
 - [x] the **latest** version of the language server (below)

### Extension

```
v2.22.0
```

### Language Server

```
0.27.0
```

### VS Code

```
Version: 1.66.2 (Universal)
Commit: dfd34e8260c270da74b5c2d86d61aee4b6d56977
Date: 2022-04-11T07:49:20.994Z
Electron: 17.2.0
Chromium: 98.0.4758.109
Node.js: 16.13.0
V8: 9.8.177.11-electron.0
OS: Darwin arm64 21.4.0
```

### Operating System

```
macOS Montery 12.3.1
```

### Terraform Version

```
Terraform v1.1.9
on darwin_arm64
```

### Steps To Reproduce

1. Open *.tf file in VSCode
2. Open command palette with `Cmd + Shift + P`
3. Type `Format Document` and click Enter
4. Get errror `There is no formatter for 'hcl' files installed.`

Include any relevant Terraform configuration or project structure:

**learn-terraform-aws-get-started/output..tf**
```terraform
output "instance_id" {
  description = "ID of the EC2 instance"
  value       = aws_instance.app_server.id
}

output "instance_public_ip" {
  description = "Public IP address of the EC2 instance"
      value       = aws_instance.app_server.public_ip
}
```

**.vscode/settings.json**
```json
{
    "editor.formatOnSave": true,
    "[hcl]": {
      "editor.defaultFormatter": "hashicorp.terraform",
      "editor.formatOnSave": false,
      "editor.codeActionsOnSave": {
        "source.formatAll.terraform": true
      },
    },
    "[terraform]": {
      "editor.defaultFormatter": "hashicorp.terraform",
      "editor.formatOnSave": false,
      "editor.codeActionsOnSave": {
        "source.formatAll.terraform": true
      },
    },
    "[terraform-vars]": {
      "editor.defaultFormatter": "hashicorp.terraform",
      "editor.formatOnSave": false,
      "editor.codeActionsOnSave": {
        "source.formatAll.terraform": true
      },
    },
    "editor.tabSize": 2,
}
```

**Output command `tree`**
```

.
├── .git
│   ├── ...
├── .gitignore
├── .vscode
│   ├── extensions.json
│   └── settings.json
└── learn-terraform-aws-get-started
    ├── .terraform
    │   ├── providers
    │   │   └── registry.terraform.io
    │   │       └── hashicorp
    │   │           └── aws
    │   │               └── 3.75.1
    │   │                   └── darwin_arm64
    │   │                       └── terraform-provider-aws_v3.75.1_x5
    │   └── terraform.tfstate
    ├── .terraform.lock.hcl
    ├── main.tf
    ├── output.tf
    └── tmp
        ├── terraform.tfstate
        └── terraform.tfstate.backup

46 directories, 59 files
```

### Expected Behavior
Format document with `terraform fmt`

### Actual Behavior
Error message `There is no formatter for 'hcl' files installed`

**Comment by fullheart** [comment]
Hi @jpogran thank you for the fast reply. Just yet I saw, that I installed also this extension: [`wholroyd.HCL`](https://marketplace.visualstudio.com/items?itemName=wholroyd.HCL). After uninstalling this extension I didn't get anymore the mentioned error message. 
And after I revert also my experimental VSCode settings (e.g. direct path to `terraform-ls`) to workaround the problem and reinstalling this extension, the `Format Document` command works again.

Sorry, for my "wrong alarm". Thank you for your help!
